### ISSUE TITLE
CRLF at the end of the cell gets trimmed

### ISSUE THREAD

**Issue by spivachek** [paulfitz/daff#95]
Hi! I've encountered the situation where CRLF at the end of the cell value gets trimmed when output is done as plain text csv (without rendering). With rendering CRLF is kept. Examples:

Kept with rendering:
```
$ daff diff <(echo text) <(echo -e "text\n\"\r\n\"")
@@ ,text
+++,"
"
```
But trimmed without:
```
$ daff --no-color diff <(echo text) <(echo -e "text\n\"\r\n\"")
@@,text
+++,""
```
This occurs in Csv.hx in renderCell function between lines 126-134. \r, \n are first stored into line_buf and later appended to result. But when cell text ends with \r, \n they don't get appended. Is this on purpose?

**Comment by paulfitz** [comment]
Oh my gosh you're right, that's bad.  Sorry I missed this issue for over a year :blush:.  Working on a fix in #117.

**Comment by paulfitz** [comment]
Should be fixed in 1.3.34.
